collectors/logsmanagement.plugin/plugin_logsmanagement_systemd.c

```diff
@@ -0,0 +1,204 @@
+        chart_data->dim_facil[7] = rrddim_add(chart_data->st_facil, "7:news/nntp", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);     
+        chart_data->dim_facil[8] = rrddim_add(chart_data->st_facil, "8:uucp", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);  
+        chart_data->dim_facil[9] = rrddim_add(chart_data->st_facil, "9:time", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_facil[10] = rrddim_add(chart_data->st_facil, "10:sec/auth", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_facil[11] = rrddim_add(chart_data->st_facil, "11:ftp", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_facil[12] = rrddim_add(chart_data->st_facil, "12:ntp", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_facil[13] = rrddim_add(chart_data->st_facil, "13:logaudit", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_facil[14] = rrddim_add(chart_data->st_facil, "14:logalert", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_facil[15] = rrddim_add(chart_data->st_facil, "15:clock", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);                                                 
+        chart_data->dim_facil[16] = rrddim_add(chart_data->st_facil, "16:local0", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);     
+        chart_data->dim_facil[17] = rrddim_add(chart_data->st_facil, "17:local1", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL); 
+        chart_data->dim_facil[18] = rrddim_add(chart_data->st_facil, "18:local2", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL); 
+        chart_data->dim_facil[19] = rrddim_add(chart_data->st_facil, "19:local3", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL); 
+        chart_data->dim_facil[20] = rrddim_add(chart_data->st_facil, "20:local4", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL); 
+        chart_data->dim_facil[21] = rrddim_add(chart_data->st_facil, "21:local5", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL); 
+        chart_data->dim_facil[22] = rrddim_add(chart_data->st_facil, "22:local6", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_facil[23] = rrddim_add(chart_data->st_facil, "23:local7", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL); 
+        chart_data->dim_facil[24] = rrddim_add(chart_data->st_facil, "uknown", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);   
+    }
+
+
+}
+
+void systemd_chart_collect(struct File_info *p_file_info, struct Chart_meta *chart_meta){
+    chart_data_systemd_t *chart_data = chart_meta->chart_data_systemd;
+
+    /* Number of lines - collect */
+    chart_data->num_lines_total = p_file_info->parser_metrics->num_lines_total;
+    chart_data->num_lines_rate += p_file_info->parser_metrics->num_lines_rate;
+    p_file_info->parser_metrics->num_lines_rate = 0;
+
+    /* Syslog priority value - collect */
+    if(p_file_info->parser_config->chart_config & CHART_SYSLOG_PRIOR){
+        for(int j = 0; j < 193; j++){
```

Comment:
What I wrote about `192` also applies for this `193` here.

Reply:
and for next, values, should we next an enum for the smaller values? :thinking:

Reply:
Done, all magic numbers must have been fixed now.